### DISCUSSION TITLE
Automatic Segmentation using keyframes similar to Scene Edit Detection in Premiere Pro

### DISCUSSION THREAD

**Discussion by xcodelight** [mifi/lossless-cut#1396] [Ideas]
Hello
First of all. Thanks for this amazing software that you have created. It is very useful for me. 

I'm sure you know about the feature in premier pro which is Scene Edit Detection which allows you to automatically add cut points to a scene. 
In lossless-cut , you have already added keyframe detection. Can you add a feature which generate automatic Segmentation using keyframes? That would be really useful and timesaving .

**Comment by mifi** [comment]
that's a great idea. creating a feature request from this #1398

**Reply by xcodelight** [reply]
Thank you. Can't wait for this feature to come out
